2024/day02/solution.md

```
@@ -0,0 +1,70 @@
+## Basic linux commands
+
+### Listing commands
+```ls option_flag arguments ```--> list the sub directories and files avaiable in the present directory
```

Markdownlint

Spaces inside code span elements

(MD038, no-space-in-code)